pkg/apiserver/api/v1/instance.proto

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2019 The KubeCarrier Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+syntax = "proto3";
+package kubecarrier.api.v1;
+option go_package = "v1";
+
+import "google/api/annotations.proto";
+import "google/protobuf/empty.proto";
+
+import "meta.proto";
+
+message Instance {
+  ObjectMeta metadata = 1;
+  string spec = 2;
+  string status = 3;
```

Comment:
This needs more information to be useful:
* maybe reference to the offering? it would satisfy most requirements since I could easily fetch the offering and find out the details
* instance image, provider, readiness status (boolean).

Reply:
UI can get corresponding `offering` using `instance`

Reply:
Add offering information in the instance.

Reply:
done